ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js/tiny-slider.js

```diff
@@ -0,0 +1,3178 @@
+var tns = (function (){
```

Comment:
@Rajamouli-hcl : this tiny-slider.js should go in clientlib-site and not in component clientlib

Reply:
Updated as per comments!